test/remote/gateways/remote_adyen_test.rb

```diff
@@ -246,62 +246,6 @@ def test_successful_authorize_with_3ds_dynamic_rule_broken
     assert_equal response.params['resultCode'], 'Authorised'
   end
 
-  def test_successful_purchase_with_auth_data_via_threeds1_standalone
-    eci = '05'
-    cavv = '3q2+78r+ur7erb7vyv66vv\/\/\/\/8='
-    cavv_algorithm = '1'
-    xid = 'ODUzNTYzOTcwODU5NzY3Qw=='
-    directory_response_status = 'Y'
-    authentication_response_status = 'Y'
-    options = @options.merge(
-      three_d_secure: {
-        eci: eci,
-        cavv: cavv,
-        cavv_algorithm: cavv_algorithm,
-        xid: xid,
-        directory_response_status: directory_response_status,
-        authentication_response_status: authentication_response_status
-      }
-    )
-
-    auth = @gateway.authorize(@amount, @credit_card, options)
-    assert_success auth
-    assert_equal 'Authorised', auth.message
-    assert_equal 'true', auth.params['additionalData']['liabilityShift']
-
-    response = @gateway.purchase(@amount, @credit_card, options)
-    assert_success response
-    assert_equal '[capture-received]', response.message
-  end
-
-  def test_successful_purchase_with_auth_data_via_threeds2_standalone
-    version = '2.1.0'
-    eci = '02'
-    cavv = 'jJ81HADVRtXfCBATEp01CJUAAAA='
-    ds_transaction_id = '97267598-FAE6-48F2-8083-C23433990FBC'
-    directory_response_status = 'C'
-    authentication_response_status = 'Y'
-
-    options = @options.merge(
-      three_d_secure: {
-        version: version,
-        eci: eci,
-        cavv: cavv,
-        ds_transaction_id: ds_transaction_id,
-        directory_response_status: directory_response_status,
-        authentication_response_status: authentication_response_status
-      }
-    )
-
-    auth = @gateway.authorize(@amount, @credit_card, options)
-    assert_success auth
-    assert_equal 'Authorised', auth.message
-    assert_equal 'true', auth.params['additionalData']['liabilityShift']
-
-    response = @gateway.purchase(@amount, @credit_card, options)
-    assert_success response
-  end
-
```

Comment:
Should these tests instead be improved (not as part of this commit)? It doesn't seem there are any other remote tests that support the 3DS auth data fields.

Reply:
🤔 Probably worth adding back, I am happy to revert the deletion, but I hate to see failing tests in master.

Reply:
Or instead commented-out with the note that they require certain account capabilities?

Reply:
I'd opt for removal over comment out since the only way to run them would be to uncomment them :)

Reply:
Commenting out remote tests that only work _sometimes_ is a pattern I occasionally see in Active Merchant. Example: https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_worldpay_test.rb

Would this be a matter of getting Spreedly's test account the right capabilities in order for it to pass?

Reply:
We support Adyen 3DS auth data fields, so we should have (working) tests for them.

Reply:
I'll let them fail for now, and circle back in another PR and fix them.